docs/full-text-search.md

```diff
@@ -0,0 +1,47 @@
+# Full-text search
+
+Zulip supports full-text search. By default, it supports only
+English.
+
+TODO: More description.
+
+## The default full-text search implementation
+
+Zulip uses
+[PostgreSQL built-in full-text search feature](http://www.postgresql.org/docs/current/static/textsearch.html).
+
+TODO: More description.
+
+## How to enable full-text search against all languages
+
+Zulip also supports [PGroonga](http://pgroonga.github.io/). PGroonga
+is a PostgreSQL extension that provides full-text search
+feature. PGroonga supports all languages including Japanese, Chinese
+and so on.
+
+This section describes how to enable full-text search feature based on
+PGroonga.
+
+First, you [install PGroonga](http://pgroonga.github.io/install/).
+
+Then, you grant `USAGE` privilege on `pgroonga` schema to `zulip`
+user:
+
+    GRANT USAGE ON SCHEMA pgroonga TO zulip;
```

Comment:
Yes when we decide how to provide the PGroonga setup script. It will be discussed at https://github.com/zulip/zulip/pull/700/files/3532f455a1bff2fb8aa413cceaa6b1aba9fdb8de#r60951735 or new issue.